### ISSUE TITLE
It can not be started

### ISSUE THREAD

**Issue by dousee163** [expo/expo-cli#1896]
It stopped at
Opening DevTools in the browser... (press shift-d to disable)

**Comment by byCedric** [comment]
Hi @dousee163! Unfortunately, this is not enough info for us to determine the issue. The Metro bundler can sometimes take a few minutes to start up. But there are some things you can try, like:

- Remove the `.expo` folder
- Clear the metro bundler cache with `expo start --clear` 
- Or start expo in debug mode `EXPO_DEBUG=true expo start`

If this still doesn't work for you, open a new issue with the information filled in. Optionally, you can add the exact output of `EXPO_DEBUG=true expo start` to that issue so we can help you better 😄 

Hope it helps!

**Comment by BettyJJ** [comment]
@byCedric  I have the same problem. It all worked perfectly before. About 4 or 5 days ago it suddenly stopped working. I have tried `EXPO_DEBUG=true` but no extra info is displayed. It just stopped at 
```
Expo DevTools is running at http://localhost:19002
Press d to open DevTools now, or shift-d to always open it automatically.
```
Result of `expo diagnostics`:
```
  Expo CLI 3.18.6 environment info:
    System:
      OS: Windows 10 10.0.17763
    Binaries:
      Node: 10.15.3 - D:\nodejs\node.EXE
      npm: 6.4.1 - D:\nodejs\npm.CMD
    npmPackages:
      expo: 36.0.0 => 36.0.0
      react: 16.9.0 => 16.9.0
      react-native: https://github.com/expo/react-native/archive/sdk-36.0.0.tar.gz => 0.61.4
      react-navigation: 3.11.1 => 3.11.1
```
I wonder if @dousee163 is from China and this has something to do with the network condition here.  @byCedric Can you tell us if the metro bundler needs to access internet when it starts?

**Comment by byCedric** [comment]
Hi @BettyJJ, thanks for the info! I don't think the Metro bundler itself requires internet. We do require internet if you want to publish your project, to authenticate you on Expo. Maybe it could be blocked in China somehow? You can try starting in offline mode with `expo start --offline`, that should disable the Expo services for you locally.

**Comment by BettyJJ** [comment]
@byCedric Thank you very much! 

Starting with `expo start --offline` works. But as you said, we can't publish the project in this way.

I think this problem happened because expo's server was blocked here. I tried pinging expo.io and exp.host (both resolved to 104.197.216.164) in the past few days and all failed. However just now I tried again and pinged successfully and was able to start and publish the project. They coincide. 

Since the block seems to be lifted now, I hope everything continues to be OK.
